core/src/main/scala/chisel3/core/package.scala

```diff
@@ -66,13 +66,13 @@ package object core {
   type RawModule = chisel3.RawModule
   @deprecated("Avoid importing from chisel3.core, these are not public APIs and may change at any time. " +
     "Use chisel3.MultiIOModule instead. This alias will be removed in 3.4.", "since the beginning of time")
```

Comment:
Do these docs need to be updated?

Reply:
Yes, but they are just deleted by https://github.com/chipsalliance/chisel3/pull/1730 so I'll leave it alone to not conflict.